src/Tasks/Microsoft.NET.Build.Tasks.UnitTests/GivenAnAppHost.cs

```diff
@@ -0,0 +1,255 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Text;
+using FluentAssertions;
+using Xunit;
+
+namespace Microsoft.NET.Build.Tasks.UnitTests
+{
+    public class GivenAnAppHost
+    {
+        private const string _placeHolder = "c3ab8ff13720e8ad9047dd39466b3c8974e592c2fa383d4a3960714caef0c4f2"; //hash value embedded in default apphost executable
+        private readonly static byte[] _bytesToSearch = Encoding.UTF8.GetBytes(_placeHolder);
+
+        [Fact]
```

Comment:
Sorry - I don't understand your comment.
The GUI bit only works on Windows RIDs. I guess it might be possible to run the publish on Mac for a Windows RID and it should still work... I'll have to try that somehow.

Reply:
nvm, it is not real file